Corrige des erreurs de typographie

rezemika opened the pull request:
Ce commit remplace les espaces par des espaces insécables quand c'est pertinent (c.a.d. avant les double points, les points d'interrogation, les points d'exclamation et les points-virgules).
Il replace aussi les ponctuations de fin de phrases à l'intérieur des blocs de traduction quand cela est possible.

En conséquence, il peut être nécessaire de vérifier les fichiers de traduction générés avec 'makemessages' (pour les instances qui les utilisent).

| Q                                   | R
| ----------------------------------- | -------------------------------------------
| Type de modification                | correction de bug
| Ticket(s) (_issue(s)_) concerné(s)  | #4584 

**J'ai vérifié le diff et le fonctionnement de mon instance locale, mais il est fort possible que des erreurs m'aient échappées.**

### QA

- Vérifier le diff (risque d'être long et fastidieux).
- Vérifier le bon fonctionnement des vues sur une instance locale.
- Vérifier le bon affichage des phrases.

motet-a commented:
~~Travis est content sur https://github.com/motet-a/zds-site/tree/pull-4620.~~~

C’était un problème d’espaces insécables dans `zds.member.tests.tests_views`.

motet-a commented:
En fait, je me demande si c’est vraiment une bonne chose de mettre des échappements HTML comme `&nbsp;` dans les blocs `{% trans … %}`. Ne vaudrait-t-il pas mieux mettre directement de l’Unicode ?

rezemika commented:
Tu as surement raison. À vrai dire, j'ai utilisé une regex et je n'avais pas pensé à ce cas là.

motet-a commented:
Je doute pas mal quand-même. Je pense qu’il faudrait avoir l’avis de quelqu’un qui connais bien Django et compagnie.

motet-a commented:
Bon, comme il a été décidé de remplacer tous les `&nbsp;` par des vrais espaces insécables, j’ai modifié cette PR.

gllmc commented:
@rezemika : j'ai relu et tout ça me semble bon. Par contre, je ne sais pourquoi Travis n'est pas passé, essaye de faire un rebase.

rezemika committed:
Corrige des erreurs de typographie

Ce commit remplace les espaces par des espaces insécables quand c'est
pertinent (c.a.d. devant les double points, les points
d'interrogation, les points d'exclamation et les points-virgules). Il
replace aussi les ponctuations de fin de phrases à l'intérieur des
blocs de traduction quand cela est possible.

En conséquence, il peut être nécessaire de vérifier les fichiers de
traduction générés avec 'makemessages'.

motet-a commented:
Bon allez, on ne va pas faire traîner ça plus que ça :upside_down_face:

motet-a commented:
Merci @rezemika !
